BouncyCastle/src/CCMBlockCipher.crysl

```diff
@@ -0,0 +1,31 @@
+SPEC org.bouncycastle.crypto.modes.CCMBlockCipher
+
+OBJECTS
+	org.bouncycastle.crypto.BlockCipher engine;
+	org.bouncycastle.crypto.CipherParameters params;
+
+EVENTS
+	Cons: CCMBlockCipher(engine);
```

Comment:
The naming scheme should be con (or con1, con2, ...) for single constructors with cons for the set of constructors